components/pricing-cards/index.js

```
@@ -0,0 +1,99 @@
+
+      <div className="pricing-card__body">
+        {attrs.display_description__limio && (
+          <p className="pricing-card__description">
+            {attrs.display_description__limio}
+          </p>
+        )}
+
+        {attrs.display_price__limio && (
+          <div
+            className="pricing-card__price"
+            dangerouslySetInnerHTML={{ __html: sanitiseHTML(attrs.display_price__limio) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)